civis/tests/test_io.py

```diff
@@ -145,6 +145,19 @@ def test_dataframe_to_civis(self, *mocks):
         assert result.state == 'succeeded'
 
     @patch(swagger_import_str, return_value=civis_api_spec)
+    def test_civis_multifile_unload(self, *mocks):
+        sql = "SELECT * FROM scratch.api_client_test_fixture"
+        result = civis.io.civis_multifile_unload(sql,
+                                                 database='redshift-general')
+        assert set(result.keys()) == {'files', 'query', 'header'}
+        assert result['query'] == sql
+        assert result['header'] == ['a', 'b', 'c']
+        assert isinstance(result['files'], list)
+        assert set(result['files'][0].keys()) == {'id', 'name', 'size', 'url'}
```

Comment:
For the same reason, the keys will be `{'id', 'name', 'size', 'url', 'url_signed'}`